app/views/patients/_menu.html.erb

```diff
@@ -1,22 +1,19 @@
 <section class="abortion-left-menu">
-  <ul class="nav nav-pills nav-stacked">
-    <li class='active'><%= link_to 'Patient Information', '#patient_information', data: { toggle: 'tab' } %></li>
-    <li><%= link_to 'Abortion Information', '#abortion_information', data: { toggle: 'tab' } %></li>
-    <li><%= link_to 'Notes', '#notes', data: { toggle: 'tab' } %></li>
-    <li><%= link_to 'Change Log', '#change_log', data: { toggle: 'tab' } %></li>
-    <li><%= link_to 'Call Log', '#call_log', data: { toggle: 'tab' } %></li>
-    <% if ( current_user.admin? || current_user.data_volunteer? ) && patient.pledge_sent == true %>
-      <li><%= link_to 'Pledge Fulfillment', '#fulfillment', data: { toggle: 'tab' } %></li>
-    <% end %>
+  <ul class="nav nav-pills nav-stacked" id='menu-list'>
+    <%= render 'menu_item', link_for: 'patient_information', active: true %>
```

Comment:
Moved this to a partial so we can more surgically rerender this, instead of flattening it.